src/test/java/edu/ucla/library/prl/harvester/InstitutionRequestsFT.java

```diff
@@ -61,6 +61,9 @@ public class InstitutionRequestsFT {
 
     private static final UriTemplate INSTITUTIONS = UriTemplate.of("/institutions");
 
+    private static final String MISSING_NAME_VALIDATION_ERROR =
```

Comment:
Why is this a local constant rather than a value in `prl-harvester_messages`?

Reply:
will move

Reply:
Moved